coral-hive/src/test/java/com/linkedin/coral/hive/hive2rel/TestUtils.java

```diff
@@ -64,12 +68,11 @@ public IMetaStoreClient getMetastoreClient() throws HiveException, MetaException
     }
   }
 
-  public static TestHive setupDefaultHive() throws IOException {
+  public static TestHive setupDefaultHive(HiveConf conf) throws IOException {
     if (hive != null) {
       return hive;
     }
-    System.out.println(System.getProperty("java.io.tmpdir"));
```

Comment:
Could we keep similar log in new implementation in case a manually look into the directory is needed?

Reply:
Thanks for reminding, added back